installer/scripts/npm_startup.sh

```diff
@@ -0,0 +1,63 @@
+cmd="omsagent ALL=(ALL) NOPASSWD: /opt/microsoft/omsconfig/Scripts/NPMAgentBinaryCap.sh"
+scriptPath="/opt/microsoft/omsconfig/Scripts/NPMAgentBinaryCap.sh"
+
+SudoSupportsIncludeDirective() {
+    # Algorithm:
+    #   If    '#includedir /etc/sudoers.d' exists in /etc/sudoers AND /etc/sudoers.d exists,
+    #   Then  Use /etc/sudoers.d
+    #   Else  Append to /etc/sudoers
+
+    INCLUDEDIR=0
+    egrep -q "^#includedir\s+/etc/sudoers.d" /etc/sudoers && INCLUDEDIR=1
+
+    if [ $INCLUDEDIR -eq 1 -a -d /etc/sudoers.d ]; then
+        return 0
+    else
+        return 1
+    fi
+}
+
+SudoSupportsIncludeDirective
+if [ $? -eq 0 ]; then
```

Comment:
In the RemoveSudoersSupport in uninstall script, we use the comment "# Begin sudo configuration for omsagent" and "# End sudo configuration for omsagent" to identity the section for OMS agent. Since your new line is out of the scope, it won't be removed when uninstall the omsagent package.